README.md

```diff
@@ -66,6 +69,13 @@ If you are still having issues or if your command prompt is throwing an error th
 6. Run: `pipenv run flask run`
 7. Login using one of the accounts below and you should be good to go!
 
+
+### Database commands. 
+- Create and Seed the Database: `pipenv run manage.py create`
+- Seed the Database: `pipenv run manage.py recreate`
```

Comment:
Does recreate seed the database?

Reply:
I believe so. Let me double check

Reply:
I've checked on this. `recreate` will drop, create, and seed db.
All manage.py commands need to be prefixed with `pipenv run python`

Reply:
you are correct...